packages/cms/src/components/fields/ProfileSearch.jsx

```diff
@@ -238,6 +238,9 @@ class ProfileSearch extends Component {
       profileTitles
     } = this.props;
 
+    let dimensionResults = [];
+    if (mode === "dimension" && formatResults && results && results.results) dimensionResults = formatResults(results.results);
+
```

Comment:
How would you feel about moving this usage of `formatResults` directly into the axios get? We could even make the default value be `resp => resp` so that we could just run it on the payload regardless.

Reply:
Done